README.md

```diff
@@ -1 +1,40 @@
-# HiveMind
\ No newline at end of file
+# HiveMind
+
+The HiveMind is the embedded application that runs on SwarmUS HiveBoard and uses the HiveSight
+
+## Building
+
+Note that you need arm-gcc-none-eabi available in your path, you can find it [here](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads)
+
+```
+mkdir build
+cd build
+cmake -D CMAKE_BUILD_TYPE=Debug ..
+make
+```
+
+If you want to build for the embedded target, use the toolchain on your cmake build.
+
+```
```

Comment:
I'm not sure what you mean, if you set the toolchain, cmake consider you are cross compiling. This allows the user to provide it's own compilers and flags if needed. CMAKE_TOOLCHAIN_FILE is rarely set in the cmake script from what I've seen. 

The path should be ok since you are in the build directory, so you need to go up a level. If that wasn't clear I can edit the readme, but you can see it from line 10 to 13.